src/components/items/productId/Line.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import styles from './Line.module.css';
+
+interface LineProps {
+  marginTop?: number;
+  marginBottom?: number;
+}
+
+const Line: React.FC<LineProps> = ({
+  marginTop = 0,
+  marginBottom = 0
+}) => {
+  return (
+    <hr style={{ marginTop: marginTop, marginBottom: marginBottom }} className={styles['line']} />
```

Comment:
P3:
아래 코드도 동일 동작합니다~
```suggestion
    <hr style={{ marginTop, marginBottom }} className={styles['line']} />
```